src/components/infoBox/InfoBox.styled.ts

```diff
@@ -8,7 +8,7 @@ export const InfoBoxStyledContainer = styled.div`
 	background: rgba(155, 155, 155, 0.4);
```

Comment:
Tu i poniżej magiczne kolory, które nikomu nic nie mówią. Możesz dodać je do theme z nazwą odpowiadającą temu kolorowi (nie jest to primary, secondary, wiec moze nazywac sie np lightgray albo bluesea czy cokolwiek)